docs/content/en/customize_dojo/user_management/pro_permissions_overhaul.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "🟧 Set Permissions in Pro"
+description: "Overhaul, pro feature"
+weight: 3
+---
+
+Permissions in <span style="background-color:rgba(242, 86, 29, 0.3)">DefectDojo Pro</span> have been simplified, to make it easier to assign object access.  This feature can be accessed through the [Beta UI](/en/about_defectdojo/ui_pro_vs_os/).
+
+### Opening the Permissions window 
+
+![image](images/pro_permissions.png)
+
+When looking at Product Type or Product, you can open the Permissions window to set permissions quickly.  This menu can be found in a Table by clicking the horizontal dots **"⋮"**.  IF looking at an individual **Product** or **Product Type** page, this menu can be found under the blue gear ‘⚙️’.
```

Comment:
Is there a reason for when you use double quotes versus single quotes versus backticks?
"⋮" vs ‘⚙️’